mariel/day-01/day-01-part1.js

```diff
@@ -0,0 +1,23 @@
+import fs from "fs";
+import path from "path";
+
+const __dirname = path.resolve();
+
+const day01 = () => {
+  const input = fs
+    .readFileSync(path.join(__dirname, "day-01-input.txt"), "utf-8")
+    .split("\n")
+    .map(Number);
+
+  let count = 0;
+
+  for (let i = 0; i < input.length; i++) {
+    if (input[i] > input[i - 1]) {
```

Comment:
On the first iteration of this loop this will try to access `input[-1]` which is `undefined` and that should never be less than (or greater than) any number but I was confused about why this works when I first looked at it.

There are a couple different ways you might skip the unnecessary check when you're looking at the very first element (and don't have a previous one to compare to) - you might explore some of those options. (Happy to discuss if you're looking for ideas).